cirq/experiments/__init__.py

```diff
@@ -30,6 +30,13 @@
     xeb_fidelity,
 )
 
+from cirq.experiments.random_quantum_circuit import (
+    GMON_EASY_PATTERN,
+    GMON_HARD_PATTERN,
+    GmonLayer,
+    random_quantum_circuit,
```

Comment:
As suggested by @Strilanc above, we should name this more specific. This will also make the module future-proof by preventing name collisions. Example name for the function: `make_layered_random_circuit`.

Reply:
I renamed it `random_rotations_between_grid_interaction_layers_circuit`.